packages/page-bounties/src/BountyInitiateVoting.tsx

```diff
@@ -0,0 +1,113 @@
+  index: number;
+  proposals?: DeriveCollectiveProposal[];
+}
+
+const BOUNTY_METHODS = ['approveBounty', 'closeBounty'];
+
+function BountyInitiateVoting ({ index, proposals }: Props): React.ReactElement<Props> | null {
+  const { t } = useTranslation();
+  const { api } = useApi();
+  const { isMember, members } = useMembers();
+  const [isOpen, toggleOpen] = useToggle();
+  const [accountId, setAccountId] = useState<string | null>(null);
+  const [threshold, setThreshold] = useState<BN>();
+
+  useEffect((): void => {
+    members && setThreshold(
+      new BN(Math.ceil(members.length * getTreasuryProposalThreshold(api)))
+    );
+  }, [api, members]);
+
+  const approveBountyProposal: SubmittableExtrinsic<'promise'> = (api.tx.bounties || api.tx.treasury).approveBounty(index);
+  const closeBountyProposal: SubmittableExtrinsic<'promise'> = (api.tx.bounties || api.tx.treasury).closeBounty(index);
+
+  const isVotingInitiated = useMemo(() => proposals?.filter(({ proposal }) => BOUNTY_METHODS.includes(proposal.method)).length !== 0, [proposals]);
+
+  return isMember && !isVotingInitiated
+    ? (
+      <>
+        <Button
+          icon='plus'
+          isDisabled={false}
+          label={t<string>('Initiate Voting')}
+          onClick={toggleOpen}
+        />
+        {isOpen && (
+          <Modal
+            header={t<string>('Initiate Voting')}
+            size='large'
+          >
+            <Modal.Content>
+              <Modal.Columns>
+                <Modal.Column>
+                  <p>{t<string>('This action will create a Council motion to either approve or reject the Bounty.')}</p>
+                </Modal.Column>
+                <Modal.Column>
+                </Modal.Column>
+              </Modal.Columns>
+              <Modal.Columns>
+                <Modal.Column>
+                  <InputAddress
+                    filter={members}
+                    help={t<string>('Select the council account you wish to use to create a motion for the Bounty.')}
+                    label={t<string>('vote with account')}
+                    onChange={setAccountId}
+                    type='account'
+                    withLabel
+                  />
+                </Modal.Column>
+                <Modal.Column>
+                  <p>{t<string>('The council member that will create a motion, submission equates to an "aye" vote for chosen option.')}</p>
+                </Modal.Column>
+              </Modal.Columns>
+            </Modal.Content>
+            <Modal.Actions onCancel={toggleOpen}>
+              <TxButton
+                accountId={accountId}
+                icon='check'
+                isDisabled={false}
+                label={t<string>('Approve')}
+                onStart={toggleOpen}
+                params={api.tx.council.propose.meta.args.length === 3
+                  ? [threshold, approveBountyProposal, approveBountyProposal.length]
+                  : [threshold, approveBountyProposal]
```

Comment:
In this case you can stick to the first only - bounties are only on current chains so doesn't need to align with older.